src/common/config/chartTypes/meter.js

```diff
@@ -12,7 +12,6 @@ export const options = () => {
     radius: '80%',
     circumference: 180,
     rotation: 270,
-    aspectRatio: 1.7,
```

Comment:
It looks weird with the default value of 1, too much white space. Maybe just reduce it slightly, ex ~1.5?

Reply:
I think 1.4 is good

Reply:
Done